src/pages/_error.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component, Fragment } from "react";
+import PropTypes from "prop-types";
+import Helmet from "react-helmet";
+import { withStyles } from "@material-ui/core/styles";
+import Typography from "@material-ui/core/Typography";
+import Link from "components/Link";
+
+const styles = {
+  root: {
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
+    marginTop: "2rem"
```

Comment:
There needs to be a bit more spacing here, this should be `4rem` instead of 2.

The font color should also be `black65` instead of solid black.